.eslintrc

```diff
@@ -1,3 +1,7 @@
 {
-    "extends": "react-app"
+    "extends": "react-app",
```

Comment:
Right now the only presets that are supported are those built directly into eslint, and those that come with your application code. I'm going to make the error messages for eslint include missing presets in the build status soon.

Other than 'react-app' are there other presets that react developers often use?